projects/mac_ai/testing/llama_cpp.py

```
@@ -105,7 +105,7 @@
 @entrypoint()
 def rebuild_image(start=True):
     base_work_dir = remote_access.prepare()
-    prepare_test(base_work_dir, utils.parse_platform("macos/llama_cpp/upstream"))
+    prepare_test(base_work_dir, utils.parse_platform("macos/llama_cpp/upstream_bin"))
```

Ruff (0.8.2)

Undefined name `prepare_test`

(F821)